### PR TITLE
Remove deprecated mixer for observability

### PR DESCRIPTION
**Pull request by shamsher31**
Removed https://preliminary.istio.io/latest/docs/tasks/observability/mixer/

Part of https://github.com/istio/istio/pull/25669 and https://github.com/istio/istio/issues/25333

> https://docs.google.com/document/d/1abYN18B8JyV64EFHtNm0uSIM4DNRX8JddJiwacNBoHg
> Istio 1.8 will be the first release of Istio that does not include support for Mixer-based integrations.
> It is expected that this plan will be executed immediately following the release branch cut of Istio 1.7 to allow for a complete release cycle of development without Mixer in the codebase.

### CODE REVIEW

File: content/en/docs/ops/common-problems/observability-issues/index.md [hunk 1]
```diff
@@ -45,32 +45,6 @@ service. Some common time synchronization systems are NTP and Chrony. This is es
...
-
-1. Mixer hands the instances to Mixer adapters for processing and backend storage.
-
-1. The backend storage systems record the metrics data.
-
-The Mixer default installations include a Prometheus adapter and the configuration to generate a [default set of metric values](/docs/reference/config/policy-and-telemetry/metrics/) and send them to the Prometheus adapter. The Prometheus adapter configuration enables a Prometheus instance to scrape Mixer for metrics.
-
-If the Istio Dashboard or the Prometheus queries don’t show the expected metrics, any step of the flow above may present an issue. The following sections provide instructions to troubleshoot each step.
-
 #### Verify Istio CNI pods are running (if used)
```
Comment (frankbu) [inline content/en/docs/ops/common-problems/observability-issues/index.md:48]: This looks strange now with `####` header, when the others are all `##`. This used to be nested under the Mixer section. Is it still a common problem, without Mixer? If so, what about some of the other nested entries, like `Verify Prometheus configuration`?

File: content/en/docs/ops/common-problems/observability-issues/index.md [hunk 2]
```diff
@@ -82,170 +56,3 @@ The Istio CNI plugin performs the Istio mesh pod traffic redirection in the Kube
     {{< /text >}}
 
 1. If `PodSecurityPolicy` is being enforced in your cluster, ensure the `istio-cni` service account can use a `PodSecurityPolicy` which [allows the `NET_ADMIN` and `NET_RAW` capabilities](/docs/ops/deployment/requirements/)
```
Comment (frankbu) [inline content/en/docs/ops/common-problems/observability-issues/index.md:58]:
```suggestion
1. If `PodSecurityPolicy` is being enforced in your cluster, ensure the `istio-cni` service account can use a `PodSecurityPolicy` which [allows the `NET_ADMIN` and `NET_RAW` capabilities](/docs/ops/deployment/requirements/).
```

File: content/en/docs/ops/common-problems/observability-issues/index.md (hunk 1 above)
Comment (shamsher31) [reply content/en/docs/ops/common-problems/observability-issues/index.md:48]: The `Verify Prometheus configuration` depends on `targets: ['istio-mixer.istio-system:42422']` which uses mixer in some way. So it needs to be removed.

**Comment by shamsher31** [comment]
@frankbu Lint fails because of a broken link in the `0.5` release announcement. Is that expected. Can I remove that link?
```
http://localhost:5351/news/releases/0.x/announcing-0.5/
  [404] http://localhost:5351/docs/tasks/observability/mixer/logs/fluentd/
ERROR: Detected 1 broken links. Scanned 3249 links in 46.242 seconds.
```

**Comment by frankbu** [comment]
@shamsher31 Normally, we try to fix broken links to deleted documents by changing them to point to the doc in the archive release corresponding to the release note, or to the latest version where the doc still exists. But this is such an old release note that simply removing the link would be fine in case.

**Comment by ericvn** [comment]
@shamsher31 I am working on some similar changes as we removed some of the pulled in reference docs in https://github.com/istio/istio.io/pull/8061.
You will find that I changed broken links in news/blogs by prefixing `https:://istio.io/vx.y` to the links where x.y match the particular release in the news or 1.6 if in a blog.

**Pull request merged**